src/index.css

```diff
@@ -8,3 +8,21 @@
 * {
   box-sizing: border-box;
 }
+
+.contentEl{
+  text-align: center;
+}
+
+
+
+.answerStatusEl{
+  margin-top: 20px;
+}
+.questionEl{
+  transition: background-color 5s;
+  background-color: transparent;
+}
+
+.true{
```

Comment:
Please use a more descriptive classname like `.exam2-correct-answer`.

`.true` doesn't explain what this class is for.